pkg/apis/options/options.go

```diff
@@ -45,6 +45,7 @@ type Options struct {
 	BitbucketRepository      string   `flag:"bitbucket-repository" cfg:"bitbucket_repository"`
 	EmailDomains             []string `flag:"email-domain" cfg:"email_domains"`
 	WhitelistDomains         []string `flag:"whitelist-domain" cfg:"whitelist_domains"`
+	ExtraHeaders             []string `flag:"extra-headers" cfg:"extra_headers"`
```

Comment:
We should call the flag `extra-header` to be consistent with other slice options

Also, we should add validation to the validation package to make sure the entries are in the correct format as part of the startup for the project

What is the format btw? It's not documented, we should document that

Reply:
> We should call the flag `extra-header` to be consistent with other slice options

Ok

> Also, we should add validation to the validation package to make sure the entries are in the correct format as part of the startup for the project

+1. I took a quick look at the test suite and couldn't find one that would be a good starting point. Have any suggestions?

> What is the format btw? It's not documented, we should document that

Just the raw header as a string:
    extra_headers = [
      "FOO: BAR"
    ]

Where would be a good place to put a doc?